tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -96,7 +96,11 @@ sub run {
     set_var('QESAP_DEPLOYMENT_NAME', get_var('QESAP_DEPLOYMENT_NAME', $deployment_name));
     record_info 'Resource Group', "Resource Group used for deployment: $deployment_name";
 
-    my $provider = $self->provider_factory();
+    # saptune/mr_test tests use ssh_interactive session which requires the local
+    # ssh config. HanaSR tests do not, so avoid creating it in HanaSR tests
+    my %provider_args = ();
+    $provider_args{skip_ssh_local_config} = 1 unless (get_var('MR_TEST', 0));
+    my $provider = $self->provider_factory(%provider_args);
```

Comment:
```suggestion
    my $provider = $self->provider_factory(skip_ssh_local_config => get_var('MR_TEST', 0));
```

Reply:
Good idea, but it needs to be `skip_ssh_local_config => !get_var('MR_TEST', 0)` :)

Will do that.

Reply:
I personally like this version
```
my $provider = $self->provider_factory(skip_ssh_local_config => !get_var('MR_TEST', 0));

```